bench/src/main/scala-2/cats/collections/bench/BitSetBenchmarks.scala

```diff
@@ -44,13 +44,13 @@ class BitSetBenchmarks {
   var size: Int = _
   var indices: Array[Int] = _
 
-  var values: Array[Int] = _
+  var values: Vector[Int] = _
```

Comment:
Starting with v2.13 the compiler complains, if `Array` is passed as a vararg parameter (like `immutable.BitSet(values: _*)` ) because it involves an implicit conversion to `Seq` and hence the entire array gets copied for that.

Changing to `Vector` because it is already `Seq` so no conversion is necessary.